src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -96,32 +98,45 @@ function onClick(
   }
 }
 
-export const DEFAULT_DARK_LOGO = 'opensearch_logo_dark_mode.svg';
-export const DEFAULT_LOGO = 'opensearch_logo_default_mode.svg';
+export const DEFAULT_LOGO = 'plugins/home/assets/logos/opensearch_dashboards.svg';
+export const DEFAULT_DARK_LOGO = 'plugins/home/assets/logos/opensearch_dashboards_darkmode.svg';
+
 interface Props {
   href: string;
   navLinks$: Observable<ChromeNavLink[]>;
   forceNavigation$: Observable<boolean>;
   navigateToApp: (appId: string) => void;
   branding: ChromeBranding;
+  basePath: HttpStart['basePath'];
+  // What background is the logo appearing on; this is unrelated to theme:darkMode
```

Comment:
nit - I'm not sure it's accurate to say it's `unrelated` - it just may be set independently.